src/fides/ctl/core/config/__init__.py

```diff
@@ -206,13 +223,22 @@ def get_config(config_path_override: str = "", verbose: bool = False) -> FidesCo
             credentials_dict=config_environment_dict
         )
 
+        # This is required to set security settings from the environment
+        # if that section of the config is missing
+        if not settings.get("security"):
```

Comment:
@sanders41 this was the required fix, and more evidence we need a larger config logic refactor...here's what was happening....

When running a CLI command for the first time, it detects whether or not a config file exists. If it doesn't exist, it will create one from defaults. For some reason, this wasn't grabbing the config with the security values initialized (this must be a bug in the `get_config` logic), so it was writing out a `toml` file with no `security` section. It would then read this file in and complain that that section didn't exist (which is correct).

This really needs to get cleaned up...I'd love to tackle this after unified fidesplus, as currently we have logic (I'll take the blame for this!) spread across the CLI entrypoint, CLI utils, and the config module....it makes it really hard to track these bugs down and fix things, and is quickly getting to the point of being unmaintainable (as evidenced by the complexity of the fix)